### DISCUSSION TITLE
Notification Stuck In 'Read' Status Limbo

### DISCUSSION THREAD

**Discussion by RandomDSdevel** [community/community#22404] [New to GitHub]
<p>     There appears to be a notification that came in through my on-site notifications inbox which doesn’t appear there any more, presumably because I read it, but still registers to a certain extent as unread.  On the GitHub web site as accessed via my browser, this is making it so I don’t see the empty-notification-inbox splash screen when I’ve gotten through all the rest of my notifications.  It’s also leaving the ‘unread’ badge behind on the notification icon on the GitHub site header.  Strangely, the ‘You’re all caught up!’ splash screen shows up on your iOS app, but I do see a notification counted as being there for me, too, when I click through to filter results by repository.  If I knew what issue or pull request was behind this hiccup, I’d navigate to it hoping to fix the problem that way, but I don’t since it won’t show up in my notifications list.  What could repair this situation?</p>

**Comment by RandomDSdevel** [comment]
<p>     This extra notification stuck in limbo also makes GitHub show the link to more notifications you get when there are too many to show on the main notifications page.  It doesn’t show up when I click through to there, though, either.</p>

**Comment by Lockszmith-GH** [comment]
<p>Welcome to the club :slight_smile: I just reported on it, and found a bug in github/feedback project.</p>
<p>Here is my thread:</p>
<blockquote>
<p><a href="https://github.community/t/blue-bell-stating-you-have-unread-notifications-but-i-cant-see-or-clean-it-up/211714" class="inline-onebox">Blue bell stating "you have unread notifications" but I can't see (or clean) it up</a></p>
</blockquote>

**Reply by Lockszmith-GH** [reply]
Solutions are either a manual approach:
https://github.com/orgs/community/discussions/6874#discussioncomment-1872442

A more 'under-the-hood' approach:
https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

**Reply by Lockszmith-GH** [reply]
@MartinDelille @Friz64 check either of these solutions.

**Comment by RandomDSdevel** [comment, marked as answer]
<p>     This seems to have gotten resolved now, though I’m not sure I remember what I might’ve done to do that.  Maybe notification handling got improved?  (<em>Shrugs.</em>)</p>

**Comment by MartinDelille** [comment]
I currently have this issue again :
![Screenshot 2024-03-06 at 10 45 33](https://github.com/community/community/assets/879889/0002e88f-23fe-4502-b5ff-95bbbaa19d7f)

The unread notification list is empty though: https://github.com/notifications?query=is%3Aunread

**Reply by Friz64** [reply]
I was part of a spam gist mass-pinging everyone a few days ago. Did that also hit you? It's most likely that. At least this comment seems to suggest so: https://github.com/orgs/community/discussions/111217

**Reply by Noratrieb** [reply]
You can clear the notifications with `curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"2024-03-07T21:52:10"}'`, where `TOKEN` is a PAT with notifcation scope and the date is the current date.

**Reply by MartinDelille** [reply]
It worked! thanks!

**Reply by MartinDelille** [reply]
I indeed also received this mass pinging by the way.

**Reply by anacrolix** [reply]
That fixed it, even with the date you provided as an example.

**Reply by MartinDelille** [reply]
I think the reason is that the mass-pinging happened before that date. I had an email notification of it.
